install_config/persistent_storage/persistent_storage_aws.adoc

```diff
@@ -89,6 +89,13 @@ claims] or from pods.
 <5> This is the AWS volume that will be used.
 ====
 
+[IMPORTANT]
+====
+Changing the value of the `*fstype*` parameter after the volume has been
+formatted and provisioned is not recommended, and can result in data loss and
```

Comment:
I thought we were supposed to avoid wording things as "recommendations" and instead just tell them why something is a bad idea? 

"Changing the value of the fstype parameter after the volume has been formatted and provisioned can result in data loss and pod failure."